src/ResourceManager/KeyVault/Commands.KeyVault/Commands/RemoveAzureKeyVault.cs

```diff
@@ -62,82 +90,112 @@ public class RemoveAzureKeyVault : KeyVaultManagementCmdletBase
             ParameterSetName = RemoveVaultParameterSet,
             ValueFromPipelineByPropertyName = true,
             HelpMessage = "The location of the deleted vault.")]
+        [Parameter(Mandatory = false,
+            Position = 1,
+            ParameterSetName = InputObjectRemoveVaultParameterSet,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The location of the deleted vault.")]
         [Parameter(Mandatory = true,
-            Position = 2,
+            Position = 1,
             ParameterSetName = RemoveDeletedVaultParameterSet,
             ValueFromPipelineByPropertyName = true,
             HelpMessage = "The location of the deleted vault.")]
+        [Parameter(Mandatory = true,
+            Position = 1,
+            ParameterSetName = InputObjectRemoveDeletedVaultParameterSet,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "The location of the deleted vault.")]
         [LocationCompleter("Microsoft.KeyVault/vaults")]
         [ValidateNotNullOrEmpty()]
         public string Location { get; set; }
 
-        /// <summary>
-        /// If present, do not ask for confirmation
-        /// </summary>
-        [Parameter(Mandatory = false,
-           HelpMessage = "Indicates that the cmdlet does not prompt you for confirmation. By default, this cmdlet prompts you to confirm that you want to delete the key vault.")]
-        public SwitchParameter Force { get; set; }
-
         /// <summary>
         /// If present, operate on the deleted vault entity.
         /// </summary>
-        [Parameter(Mandatory = false,
+        [Parameter(Mandatory = true,
```

Comment:
If they provided all the parameters except -InRemovedState, it would default to the first parameter set, because the first is the default and contains all the parameters in the second.